<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicWindow</class>
 <widget class="QMainWindow" name="MusicWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>78</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: none;
font-size: 16px;
color:rgb(43, 43, 43)
}

</string>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
    <property name="icon">
     <iconset resource="res1.qrc">
      <normaloff>:/images/images/316.bmp</normaloff>:/images/images/316.bmp</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="love_button">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>80</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 290 14pt &quot;Microsoft YaHei UI Light&quot;;</string>
    </property>
    <property name="text">
     <string>收藏❤</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>941</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #999999;
    height: 8px;
    background: #B0E0E6; /* 轨道背景颜色，淡蓝色 */
    margin: 2px 0;
}

QSlider::handle:horizontal {
    background: #FFFFFF; /* 滑块颜色，白色 */
    border: 1px solid #5c5c5c;
    width: 18px;
    margin: -2px 0;
    border-radius: 3px;
}

QSlider::add-page:horizontal {
    background: #E0FFFF; /* 滑块右侧颜色，更淡的蓝色 */
}

QSlider::sub-page:horizontal {
    background: #87CEFA; /* 滑块左侧颜色，浅天蓝色 */
}
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSlider" name="sliderPosition">
       <property name="tracking">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labRatio">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"> font-family: &quot;Courier New&quot;;
    font-size: 10pt;
    font-weight: bold;
font: 18pt &quot;微软雅黑&quot;;
 text-align: center;</string>
       </property>
       <property name="text">
        <string>  进度</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>971</width>
      <height>61</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>70</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));




border:none;

</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>13</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>231</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px;</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragDropMode::InternalMove</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemove">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>78</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: none;

font-size: 16px;
color:rgb(43, 43, 43)
}
</string>
    </property>
    <property name="text">
     <string>移除</string>
    </property>
    <property name="icon">
     <iconset resource="res1.qrc">
      <normaloff>:/images/images/318.bmp</normaloff>:/images/images/318.bmp</iconset>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>130</y>
      <width>181</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>78</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: none;

font-size: 16px;
color:rgb(43, 43, 43)
}
</string>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
    <property name="icon">
     <iconset resource="res1.qrc">
      <normaloff>:/images/images/214.bmp</normaloff>:/images/images/214.bmp</iconset>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>259</x>
      <y>69</y>
      <width>491</width>
      <height>351</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>489</width>
       <height>349</height>
      </rect>
     </property>
     <widget class="QLabel" name="labPic">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>721</width>
        <height>351</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>931</width>
      <height>81</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>90</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBoxCover">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="btnCover">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res1.qrc">
          <normaloff>:/images/images/music24.png</normaloff>:/images/images/music24.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labCurMedia">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
font-size: 16px;
color:rgb(43, 43, 43)</string>
        </property>
        <property name="text">
         <string>无曲目</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>351</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="btnPlay">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="toolTip">
       <string>播放</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="res1.qrc">
        <normaloff>:/images/images/620.bmp</normaloff>:/images/images/620.bmp</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPause">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="toolTip">
       <string>暂停</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="res1.qrc">
        <normaloff>:/images/images/622.bmp</normaloff>:/images/images/622.bmp</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="btnStop">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="toolTip">
       <string>停止</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="res1.qrc">
        <normaloff>:/images/images/624.bmp</normaloff>:/images/images/624.bmp</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPrevious">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>20</y>
        <width>31</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>前一曲</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="res1.qrc">
        <normaloff>:/images/images/616.bmp</normaloff>:/images/images/616.bmp</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="btnNext">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>后一曲</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="res1.qrc">
        <normaloff>:/images/images/630.bmp</normaloff>:/images/images/630.bmp</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>351</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QDoubleSpinBox" name="doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>24</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:16px;</string>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="prefix">
       <string/>
      </property>
      <property name="suffix">
       <string> 倍速 </string>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>0.500000000000000</double>
      </property>
      <property name="maximum">
       <double>1.500000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLoop">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>17</y>
        <width>52</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>52</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:12px;</string>
      </property>
      <property name="text">
       <string>循环</string>
      </property>
      <property name="icon">
       <iconset resource="res1.qrc">
        <normaloff>:/images/images/refresh16.png</normaloff>:/images/images/refresh16.png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>12</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btnSound">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res1.qrc">
           <normaloff>:/images/images/volumn.bmp</normaloff>:/images/images/volumn.bmp</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderVolumn">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #999999;
    height: 8px;
    background: #B0E0E6; /* 轨道背景颜色，淡蓝色 */
    margin: 2px 0;
}

QSlider::handle:horizontal {
    background: #FFFFFF; /* 滑块颜色，白色 */
    border: 1px solid #5c5c5c;
    width: 18px;
    margin: -2px 0;
    border-radius: 3px;
}

QSlider::add-page:horizontal {
    background: #E0FFFF; /* 滑块右侧颜色，更淡的蓝色 */
}

QSlider::sub-page:horizontal {
    background: #87CEFA; /* 滑块左侧颜色，浅天蓝色 */
}
</string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res1.qrc"/>
 </resources>
 <connections/>
</ui>
